components/molecule/field/src/index.scss

```diff
@@ -40,6 +41,7 @@ $atomHelpText-class: '.sui-AtomHelpText';
   &-labelContainer {
     align-items: $ai-molecule-field-label-container;
     display: flex;
+    font-weight: $fw-molecule-field-label;
     margin-bottom: $mb-molecule-field-label;
```

Comment:
From where was taking this property the value before creating the default value?